### PR TITLE
Frontpage

### PR DESCRIPTION
**Pull request by DarkerTH**
Should work fine :wink:

### CODE REVIEW

**Comment by niumis** [comment]
+1 jeeee... :)

**Comment by giedrskuzins** [comment]
+1 pirmyn:)

File: Gulpfile.js [hunk 1]
```diff
@@ -12,14 +12,18 @@ var batch = require('gulp-batch');
...
     npm: './node_modules/'
 };
 
-
 gulp.task('sass', function () {
-    gulp.src(dir.assets + 'style/main.scss')
+    gulp.src([
+        dir.assets + 'style/main.scss',
+        dir.npm + 'bootstrap-slider/dist/css/bootstrap-slider.min.css',
+        dir.public_assets + 'css/**'
```
Comment (zyrukas) [inline Gulpfile.js]: **?

File: Gulpfile.js [hunk 2]
```diff
@@ -44,6 +48,8 @@ gulp.task('js', function () {
     gulp.src([
         dir.npm + 'jquery/dist/jquery.min.js',
         dir.npm + 'bootstrap-sass/assets/javascripts/bootstrap.min.js',
+        dir.npm + 'bootstrap-slider/dist/bootstrap-slider.js',
+        dir.public_assets + 'js/**'
```
Comment (zyrukas) [inline Gulpfile.js]: maybe just one *?

Comment (DarkerTH) [inline Gulpfile.js]: Maybe, berods buvo palikta **, tai nesigilinau kodel. Tuoj pafixinsiu

**Comment by zyrukas** [comment]
ootstrap-select.min.css galbut galima per npm sita isirasyt? jei ne tai bus ok

File: app/Resources/views/base.html.twig [hunk 3]
```diff
@@ -1,16 +1,49 @@
...
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+            </button>
+            <a class="navbar-brand" href="{{ path('homepage') }}">ElastiCar Logo</a>
+        </div>
+        <div id="navbar" class="collapse navbar-collapse">
+            <ul class="nav navbar-nav">
+                {% block menu %}{% endblock %}
+            </ul>
+            <form id="loginForm" class="form-inline navbar-form navbar-right" style="display:none;">
```
Comment (zyrukas) [inline app/Resources/views/base.html.twig]: inline style. Geriau naudok "hidden" klase

File: app/Resources/views/base.html.twig [hunk 4]
```diff
@@ -1,16 +1,49 @@
...
+                <a href="{{ path('homepage') }}connect/facebook" onclick="logIn()" class="btn btn-find btn-success" >Facebook</a>
+            </form>
+            <div id="loggedIn" class="form-inline navbar-form navbar-right" style="display:none;" >
```
Comment (zyrukas) [inline app/Resources/views/base.html.twig]: inline style. Geriau naudok "hidden" klase

File: Gulpfile.js (hunk 2 above)
Comment (niumis) [inline Gulpfile.js]: turetu but dvi, tada tikrina ir subdirectory

**Comment by zyrukas** [comment]
+1

**Comment by zyrukas** [comment]
test

**Pull request merged**